client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-tasks.ts

```diff
@@ -17,7 +21,12 @@ export interface GetActionUrlProps {
 	wpAdminUrl: string;
 }
 
-export const getConfirmationTasks = ( { translate }: ConfirmationTasksProps ) => {
+export const GetConfirmationTasks = ( { translate, siteSlug }: ConfirmationTasksProps ) => {
```

Comment:
Ahh yeah. I was trying to be sneaky because we can't use React hooks in non-components, but I can probably pass that data to getConfirmationTasks from the parent component instead...I'll give that a try!